docs/getting-started/users-page.asciidoc

```diff
@@ -21,10 +21,9 @@ Beneath the KPI charts are data tables, which are useful for viewing and investi
 * *All users*: A chronological list of unique user names, when they were last active, and the associated domains.
 * *Authentications*: A chronological list of user authentication events and associated details, such as the number of successes and failures, and the hostname of the last successful destination.
 * *Anomalies*: Unusual activity discovered by machine learning jobs that contain user data.
-* *Events*: Ingested events that contain the `user.name` field. You can stack by the `event.action`, `event.dataset`, or `event.module` field.
-* *External alerts*: Alerts received from external monitoring tools. You can stack by the `event.module` or `event.category` field.
+* *Events*: Ingested events that contain the `user.name` field. You can stack by the `event.action`, `event.dataset`, or `event.module` field. To view To view <<det-engine-terminology, alerts>> received from external monitoring tools, scroll down to the Events table and select *Show only external alerts* on the right.
```

Comment:
```suggestion
* *Events*: Ingested events that contain the `user.name` field. You can stack by the `event.action`, `event.dataset`, or `event.module` field. To find <<det-engine-terminology, alerts>> received from external monitoring tools, scroll down to the Events table and select *Show only external alerts* on the right.
```

Reply:
I know "find" has a different connotation than "view" so feel free to reject this edit if it's not the message you want to send.